sw/device/silicon_creator/lib/sigverify_unittest.cc

```diff
@@ -86,7 +87,48 @@ constexpr sigverify_rsa_buffer_t kEncMsg{
 constexpr std::array<uint8_t, 4> kSignedRegion{'t', 'e', 's', 't'};
 constexpr sigverify_rsa_buffer_t kSignature{};
 
-class SigVerifyTest : public mask_rom_test::MaskRomTest {
+/**
+ * Life cycle states used in parameterized tests.
+ */
+
+const std::array<lifecycle_state_t, 8> kLcStatesTest{
```

Comment:
Is there are reason why `const` is preferred over `constexpr` here and below?

Reply:
No, thanks for catching this, these should have been `constexpr`s.